target/scripts/helpers/error.sh

```diff
@@ -66,9 +66,9 @@ function dms_panic__invalid_value { dms_panic 'invalid-value' "${1}" "${2}"; }
 # `dms_panic` methods should be preferred if your failure type is supported.
 function _shutdown
 {
-  local FATAL_ERROR_MESSAGE=$1
-
-  _notify 'fatal' "${FATAL_ERROR_MESSAGE}"
+  _notify 'fatal' "${1}"
   _notify 'err' "Shutting down.."
+
   kill 1
+  exit 1
 }
```

Comment:
Residue? It seems like the changes in full were carried over to https://github.com/docker-mailserver/docker-mailserver/pull/2486 and no longer belongs here? (although you dropped the `kill 1` in the other PR)

Reply:
Done in 3c7483149d53e750925fe5651df3fadac1ae239e